_posts/2020-06-24-Join-the-BIDS-community-on-the-BrainHack-Mattermost.md

```diff
@@ -0,0 +1,32 @@
+author: Franklin Feingold
+display: true
+---
+
+# Join the BIDS community on the BrainHack Mattermost
+
+BIDS is joining the [BrainHack Mattermost](https://mattermost.brainhack.org/){:target:"_blank"}!
+
+<!--more-->
+
+We are proud to announce our home for hosting BIDS organization instant messaging: the [BrainHack Mattermost platform](https://mattermost.brainhack.org/){:target:"_blank"}! In the past few years, several BIDS channels have independently been created on Mattermost by the community. The Steering Group and Maintainers Group have choosen to recognize the BrainHack Mattermost as a member of our organizations's official communication suite. This platform will live alongside our other official communication channels: NeuroStars (forum for support questions), mailing list (announcements), Twitter (community news), and GitHub repositories (bug reports + feature requests). 
+
+We have listed below our current set of BIDS channels. We will be establishing several BIDS channels related to: BEP development, community discussions, and more! Check it out and create your own channel! Please be sure the channel name contains `bids` in the title to increase findability.
+
+[Mattermost](https://mattermost.com/){:target:"_blank"} is an open-source alternative to Slack. The user interface and experience is very similar to that on Slack. One may join a channel, make a channel, or private message.
+
+To find our `BIDS` channels, please navigate to Mattermost and click `More...` at the bottom of the `PUBLIC CHANNELS` section. A prompt will pop up enabling you to search for our `BIDS` related channels. If you have any issues, please message Franklin on Mattermost: `@franklin_`.
+
+The [BrainHack project](https://www.brainhack.org/){:target:"_blank"} seeks to bridge the neuroscience and data science communities through hackathons and unconferences. To do this, they convene unique conferences comprised of researchers from around the world with diverse backgrounds and interests to develop our next generation of open source software for neuroscience. These workshops consist of educational activities, along with plenty of development time. If you are interested in learning more please visit [their website](https://www.brainhack.org/){:target:"_blank"}, check out [their paper](https://academic-oup-com.stanford.idm.oclc.org/gigascience/article/5/1/s13742-016-0121-x/2720978){:target:"_blank"}, and take a look at a [step by step guide](https://thewinnower.com/papers/5577-a-step-by-step-guide-for-organizing-open-collaborative-brainhack-events){:target:"_blank"} for organizing a BrainHack yourself. 
+
+We look forward to seeing you on Mattermost!
+
+Current BIDS channels on Mattermost:
+- `bids_derivatives`: conversations around BIDS-Derivatives
+- `bids_eeg`: conversations around bids and eeg data
+- `bids_ieeg`: conversations around bids and ieeg data
+- `bids_meg`: conversations around bids and meg data
```

Comment:
not sure whether combining these into `bids_ephys` would be better :thinking: but we can also do that later, so this is just a thought. no need to act.

Reply:
good question to think over - perhaps a clarifying question: do you see the ephys community coming together under a centralized group or is there some divergence with a preference to stay within their particular modality? I think there is a positive to keeping them broken down - preserving modality-specific discussions that may get lost in a broader category. Though if convergence has started and it is more beneficial for the future to keep them together, that approach sounds good to me!

Reply:
I think we are getting together because we share the same tools for analysis and more ... also, we have the common ephys BEP. The benefit of combining the channels would perhaps be a more lively and integrated discussion.

I have no strong opinion about this now 🤔

Reply:
that sounds good! As I put together our proposed Mattermost channels I will incorporate this